python/mlcroissant/mlcroissant/_src/structure_graph/nodes/metadata.py

```diff
@@ -391,6 +391,8 @@ def from_jsonld(cls, ctx: Context, metadata: Json) -> Metadata:
                     f' "{constants.SCHEMA_ORG_FILE_SET(ctx)}". Got'
                     f" {distribution_type} instead."
                 )
+        if not distribution:
+            ctx.issues.add_warning("`distribution` is empty for this dataset.")
```

Comment:
I would just remove this warning as the other messages are more explicit. WDYT?

Reply:
Sure, done.